org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/js/test.js

```diff
@@ -0,0 +1,37 @@
+var translator = require('./thingTranslator');
+var q = require('q');
+
+var argv = require('optimist')
+    .usage('Usage: $0 -i [device Id] -a [access token]')
+    .demand(['i'])
+    .demand(['a'])
+    .argv;
+
+// device object used for testing purposes (this is normally populated by the runtime)      
+function Device(deviceId, accessToken) {
+    this.props = ' { "id": "' + deviceId + '", "access_token": "' + accessToken + '" }';
+    this.name = "Wink Thermostat ";
+}
+
+var device = new Device(argv.i, argv.a);
+
+// initialize the translator for testing purposes (this is normally called by the runtime)
+translator.initDevice(device);
+
+
+// Go through a sequence of test operations for the translator
+q.delay(1000)
+ .then(() => translator.turnOn())
+ .then(() => q.delay(2000))
+ .then(() => translator.setMode("heat_only"))
+ .then(() => q.delay(2000))
+ .then(() => console.log(translator.getCurrentTemperature()))
+ .then(() => q.delay(2000))
+ .then(() => translator.setCoolingSetpoint(23))
```

Comment:
We could do this effectively with two more functions in the schema. One would be a Boolean `isCelsius` and then if that is `false` then another method in the schema is `convertToCelsius` and all of the temperature methods in the translator would call that and we would make it work. Since `convertToCelsius` isn't a translator specific function, I'm thinking if there is a way to just kind of bundle that method as a standard without re-implementing it in every translator.